app/assets/javascripts/cm_admin/filters.js

```diff
@@ -44,7 +44,7 @@ var getFilteredData = function(filterType, filterValue, filterColumn=null) {
   var searchParams = window.location.search
   var searchParamshash = getParamsAsObject(searchParams)
   var tempsearchParamshash = delete searchParamshash['page']
-  if (tempsearchParamshash.length > 0) {
+  if (Object.keys(searchParamshash).length > 0) {
```

Comment:
```suggestion
  if (Object.keys(searchParamsHash).length > 0) {
```

@anbublacky - I think this is the right naming convention.